.github/workflows/release.yaml

```diff
@@ -86,6 +86,8 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           pushd bakery
+          echo "${{ secrets.SYSEXT_PRIVATE_KEY }}" > sysext.key
+          echo "${{ secrets.SYSEXT_CERTIFICATE }}" > sysext.crt
```

Comment:
I think we should publish the cert as release artifact - and maybe include it in the sysupdate definition?

Reply:
Sysupdate has an option to verify with GPG (which is recommended) but anyway, this then would have to be part of the Ignition config. I think for now we can also provision the cert via Ignition. If the cert doesn't change it sounds ok to have it in the OS but if it's expiring every now and then, having it in the OS might be more complex for managing the lifecycle?